src/amplify-migration/amplifyMigrationScript.ts

```diff
@@ -186,8 +165,14 @@ async function modifyPermalinks(repoPath: string) {
       );
       let permalinkLineTrimmed = permalinkLine.trim();
 
-      // check if permalink has quotation marks, if so, remove them
-      permalinkLineTrimmed = permalinkLineTrimmed.replace(/"/g, "");
+      // check if permalink has quotation marks at the start and at the end, and if so, remove them
+      assert(permalinkLineTrimmed.startsWith(`permalink:`));
+      let permalinkValue = permalinkLineTrimmed
+        .replace(`permalink:`, "")
+        .trim();
+      permalinkValue = permalinkValue.replace(/^"/g, "").replace(/"$/, "");
```

Comment:
nit: We could chain up the operations for `permalinkValue` to make the code cleaner like the following. `permalinkValue` can be a `const` if no value assignment afterwards. 
```js
const permalinkValue = permalinkLineTrimmed
    .replace('permalink:', '')
    .trim()
    .replace(/^"/, '')
    .replace(/"$/, '')
    .replace(/^'/, '')
    .replace(/'$/, '');
```

Reply:
sure 871801dfcfff29a369345294ce88ac7eabc99de4